pkg/metrics/exporter/export.go

```diff
@@ -26,6 +24,8 @@ type Exporter struct {
 	outputFile string
 }
 
+var globalExporter *Exporter
```

Comment:
The exporter should be defined as a interface to other system to fetch metrics via HTTP or RPC

Reply:
Done, please take a look.